tuxemon/core/states/start/__init__.py

```diff
@@ -85,9 +85,11 @@ def set_player_name(text):
 
         def new_game():
             # load the starting map
+            starting_map = prepare.CONFIG.starting_map
+            if prepare.ARGS.starting_map:
```

Comment:
remove `prepare.ARGS`.  there should be a single object to get game options from